dbtmetabase/parsers/dbt_folder.py

```diff
@@ -52,17 +52,19 @@ def read_models(
             includes = []
         if excludes is None:
             excludes = []
+        if schema is None:
+            schema = "public"
 
         if database:
             logging.info(
                 "Argument --database %s is unused in dbt_project yml parser. Use manifest parser instead.",
                 database,
             )
 
-        if docs_url:
+        if dbt_docs_url:
```

Comment:
I know this is existing code, but I missed it in the last review. This logger should be in the `export()` method, not the parser itself. The user may have gotten here via programmatic API, so the CLI flag would be meaningless to them.

Reply:
It was originally for passing lint checks since args have to be used but we want the API for both readers to be the same without using kwargs. But good point, I have moved this to warn in the init and adjusted the code to pass linting still